src/map/lapack2flash/FLASH_orgbr.c

```diff
@@ -100,11 +100,12 @@
     FLA_Part_2x1( t, &t,                                                \
                      &T, FLA_Obj_min_dim( Ah ), FLA_TOP );              \
                                                                         \
+/* TODO: I believe this should be adding a buffer and it is not*/       \
```

Comment:
As noted, I believe this should be creating T with a buffer. However it is not and causing a few issues. I'll need to circle back to this